apps/minaintyg/src/mocks/handlers.ts

```diff
@@ -121,6 +122,7 @@ export const handlers = [
             information: [],
           }),
         ],
+        texts: fakerFromSchema(certificateTextSchema)({ PREAMBLE_TEXT: 'Ingresstext' }),
```

Comment:
Could use the mock preamble text with `{PREAMBLE_TEXT: certificateIngress(certificate.id) ?? 'Ingresstext'}`